Resources/doc/reference/api.rst

```diff
@@ -32,6 +32,12 @@ Here's the configuration we used, you may adapt it to your needs:
 
     twig:
         exception_controller: 'FOS\RestBundle\Controller\ExceptionController::showAction'
+    
+    # for SonataPageBundle > 2.3.6    
```

Comment:
It is mentioned here https://sonata-project.org/bundles/page/master/doc/reference/installation.html that DatagridBundle is used for SonataPageBundle > 2.3.6. Am  I wrong?